catalog/ui/search-ui/simple/src/main/resources/SearchPage.jsp

```diff
@@ -475,5 +476,29 @@
 
  	<script type="text/javascript" src="js/Search-min.js?bust=${timestamp}"></script>
 
+    <!-- Usage Modal -->
+    <!-- Only show if the modal is enabled -->
+    <% if(props.getSystemUsageEnabled() == true) { %>
+        <!-- Do not allow users to click outside of the modal to dismiss it, the OK button must be pressed -->
+        <script>
+            jQuery(function () {
+                $('#usageModal').modal({
+                    backdrop: 'static',
+                    keyboard: false
+                })
+            })
+        </script>
+        <div id="usageModal" class="modal hide" tabindex="-1" role="dialog"
+            aria-labelledby="myModalLabel" aria-hidden="true">
+            <div class="modal-header">
+                <h3><%=props.getSystemUsageTitle()%></h3>
+            </div>
+            <div class="modal-body"><%=props.getSystemUsageMessage()%></div>
```

Comment:
❗️ If the SystemUsageMessage contains HTML tags for formatting, they will be printed along with the string instead of formatting the text.

Reply:
Should be fixed with the jsoup change